definitions/variable/afolu/tag_forestry_types.yaml

```
@@ -0,0 +1,24 @@
+- Raw products:
```

Please use capitalized tag names for consistency.
```suggestion
- Raw Products:
```